project/frontend/src/components/RestaurantCard.js

```diff
@@ -10,79 +18,205 @@ import Globe from '../assets/globe.svg';
 import Phone from '../assets/phone.svg';
 import Clock from '../assets/clock.svg';
 
+/**
+ * Returns the starting index of the nth instance of a given string
+ * within a specified larger string.
+ *
+ * @param {string} haystack The string in which to search.
+ * @param {string} needle The string for which to search.
+ * @param {number} n Specifies which instance of 'needle' to find.
+ * @return {number} The starting index of the nth 'needle'.
+ */
+function nthIndexOf(haystack, needle, n) {
+  let i = -1;
+  while (n--) {
+    i = haystack.indexOf(needle, ++i);
+    if (i < 0) break;
+  }
+  return i;
+}
+
+/**
+ * A list of nondescript tags that do not need to be displayed
+ * in the RestaurantCard component.
+ *
+ * @type {!Array<string>}
+ */
+const bannedTags = ['food', 'point_of_interest', 'establishment'];
```

Comment:
Why are these banned?

Reply:
I figured they were redundant to show on the restaurant card, since people already know they are looking at a restaurant. `Point of Interest` and `Establishment` are also just super generic so I don't think it adds anything. We can talk about changing this though if you disagree.